### PR TITLE
Add ghostty terminal theme

### PR DESCRIPTION
**Pull request by dartt0n**
This pull request adds template, builder entry, make command and compiled config theme for [Ghostty](https://ghostty.org) terminal.

The exact colors are chosen similarly to the theme for kitty or termux terminals since config is really similar.

### CODE REVIEW

**Comment by MindTooth** [comment]
Thanks you. 😁

**Comment by MindTooth** [comment]
Btw., would you be interested in submitting it to https://github.com/mbadolato/iTerm2-Color-Schemes as well? 😇

File: ghostty/srcery [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# https://github.com/srcery-colors/srcery-terminal/
+cursor-color = #fbb829
+foreground = #fce8c3
+background = #1c1b19
+selection-foreground = #1c1b19
+selection-background = #fce8c3
+
+pallete = 0=#1c1b19
+pallete = 1=#ef2f27
+pallete = 2=#519f50
```
Comment (jbpratt) [inline ghostty/srcery:10]: s/pallete/palette

https://ghostty.org/docs/features/theme#example
https://ghostty.org/docs/config/reference#palette

Comment (MindTooth) [reply ghostty/srcery:10]: I did not catch. 😔   @dartt0n something you fix?

Comment (dartt0n) [reply ghostty/srcery:10]: Oh, yeah! Sorry for the typo, it was a very late night push...

**Pull request merged**